### PR TITLE
docs/update&complete user.yaml doc

### PR DESCRIPTION
**Pull request by AmirAliEidivandi**
image

### CODE REVIEW

File: modules/user/user.yaml [hunk 1]
```diff
@@ -24,6 +24,186 @@ paths:
                 $ref: '#/components/schemas/ErrorResponse'
         '500':
           description: Internal server error
+  /resendVerification:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/resendVerification

File: modules/user/user.yaml [hunk 2]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/ResendVerificationResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /verify:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/verify

File: modules/user/user.yaml [hunk 3]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/VerifyResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /login:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/login

File: modules/user/user.yaml [hunk 4]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/LoginResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /changePassword:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/changePassword

File: modules/user/user.yaml [hunk 5]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/ChangePasswordResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /updateProfile:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/updateProfile

File: modules/user/user.yaml [hunk 6]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/UpdateProfileResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /forgetPassword:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/forgetPassword

File: modules/user/user.yaml [hunk 7]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/ForgetPasswordResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /setNewPassword:
```
Comment (ethyaan) [inline modules/user/user.yaml]: Path is wrong,
should be /user/setNewPassword

File: modules/user/user.yaml [hunk 8]
```diff
@@ -24,6 +24,186 @@ paths:
...
+                $ref: '#/components/schemas/SetNewPasswordResponse'
+        '4XX':
+          description: Invalid request body or Unprocessable Entity
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ErrorResponse'
+        '500':
+          description: Internal server error
+  /getProfile:
```
Comment (ethyaan) [inline modules/user/user.yaml]: path is wrong,
should be /user/getProfile

**Review by ethyaan — CHANGES_REQUESTED** [review]

File: modules/user/user.yaml (hunk 7 above)
Comment (AmirAliEidivandi) [reply modules/user/user.yaml]: oops, now that's ok

Comment (ethyaan) [reply modules/user/user.yaml]: great

**Review by ethyaan — APPROVED** [review]

**Pull request merged**